components/containerd/pod/cri.go

```diff
@@ -200,7 +201,12 @@ func checkContainerdRunning(ctx context.Context) bool {
 	return false
 }
 
-func listSandboxStatus(ctx context.Context, endpoint string) ([]PodSandbox, error) {
+func listAllSandboxes(
```

Comment:
Please find another way to do test..As I commented on others, if we want to mock some functions, we would define a function variable which can be replaced in tests and after test, we would reset the function variable to the real functuon.. Introduce these two new params  which lead the complex to the caller.. it is unreasonable..

Reply:
@eahydra Fixed.